avm-transpiler/bootstrap.sh

```diff
@@ -16,4 +16,7 @@ if [ -n "$CMD" ]; then
   fi
 fi
 
+# Attempt to just pull artefacts from CI and exit on success.
+./bootstrap_cache.sh && exit
```

Comment:
Optional: would it make sense to add an echo on failure along the lines of: "If you want to use the cache install docker and set up your AWS credentials"?